src/WebJobs.Script.Analyzers/Constants.cs

```diff
@@ -5,15 +5,22 @@ namespace Microsoft.Azure.Functions.Analyzers
 {
     internal static class Constants
     {
+        internal static class Assemblies
+        {
+            public const string WebJobsAssemblyName = "Microsoft.Azure.WebJobs";
+            public const string WebJobsHostAssemblyName = "Microsoft.Azure.WebJobs.Host";
+        }
+
         internal static class Types
         {
             public const string FunctionNameAttribute = "Microsoft.Azure.WebJobs.FunctionNameAttribute";
         }
 
         internal static class DiagnosticsCategories
         {
-            public const string Reliability = "Reliability";
-            public const string Usage = "Usage";
+            public const string Reliability = nameof(Reliability);
```

Comment:
nit question: it's cool to know we can use nameof here. But If someone decides to do a rename on the caller site (by doing Ctrl + R +R ) on the constant name, will it also update the value?

Reply:
It would. "Which shouldn't be a problem because the tests would break," I thought, then looked at the tests, and it turns out they don't check expected vs actual `Category`. (There's not a way to do so without adding a custom `DiagnosticVerifier` either.)

tl;dr It's cool but you make a good point. Reverting to strings.